src/app/machines/views/MachineDetails/MachineHeader/index.scss

```diff
@@ -0,0 +1,19 @@
+@mixin MachineHeader {
+  .p-table-menu {
+    margin-right: $spv--large;
+  }
+
+  // These media queries are for when the screen is too small to display the action group without wrapping.
+  // Wrapping begins before $breakpoint-small, hence the direct reference to pixel width.
+  @media screen and (max-width: 792px) {
```

Comment:
Could you please add this as a new custom breakpoint to `_settings.scss` which will help us keep track of them all in a single place. Ideally using a single breakpoint and `$breakpoint-name + 1` `$breakpoint-name - 1` convention that we use in vanilla.

Reply:
Sure, although not sure what to name it - it's closer to `small` than `large` so I don't feel like `medium` would fit, and `kinda-small` just doesn't seem right lol

Reply:
I think I'll settle on `medium-small`